mautic/mautic
# Changes from PR#10782 rebuilt

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [Yes ]
| New feature/enhancement? (use the a.x branch)      | [No]
| Deprecations?                          | [No]
| BC breaks? (use the c.x branch)        | [No]
| Automated tests included?              | [No] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #11290

#### Description:

Looks like changes introduced by https://github.com/mautic/mautic/pull/10782 and https://github.com/mautic/grapesjs-preset-mautic/pull/26 were overwritten with next GrapesJsBuilderBundle scripts build. 

In the grapesjs-preset-mautic I added changes to dist files and removed copies, which are no longer udpated and used by other scripts - https://github.com/mautic/grapesjs-preset-mautic/pull/30. Instead, the originals in the relevant subdirectories are used, e.g.
dist/dynamicContent.domcomponents.js - removed 
dist/dynamicContent/dynamicContent.domcomponents.js - in use and updated by `npm run build`

#### Steps to test this PR:

1. With Mautic branch 4.3, 4.4 or 5.x open the file `plugins\GrapesJsBuilderBundle\Assets\library\js\dist\builder.js`
2. Try to find changes introduced by https://github.com/mautic/grapesjs-preset-mautic/pull/26 e.g. `DC: ${dynamicContents.length} components found` (after building and minifying it may look like `DC: ${t.length} components found`). If nothing is found, the scripts with Dynamic Content for the MJML email are missing.
3. Checkout this PR and try to locate the changes again. You should now be able to find the scripts from https://github.com/mautic/grapesjs-preset-mautic/pull/26/files in builder.js



<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11319"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
